includes/commands/class-mu-migration-posts.php

```diff
@@ -108,19 +109,29 @@ function ( $result ) use ( &$equals_id, &$author_not_found, $ids_map, $verbose,
 					$author_not_found[] = absint( $result->ID );
 				}
 
+				// Automatically add Woocommerce user id field
 				if ( $is_woocommerce ) {
-					$old_customer_user = get_post_meta( (int) $result->ID, '_customer_user', true );
-
-					if ( isset( $ids_map->{$old_customer_user} ) && $old_customer_user != $ids_map->{$old_customer_user} ) {
-						$new_customer_user = $ids_map->{$old_customer_user};
-
-						update_post_meta( (int) $result->ID, '_customer_user', $new_customer_user );
-
-						$this->log( sprintf(
-							__( 'Updated customer_user for "%s" (ID #%d)', 'mu-migration' ),
-							$result->post_title,
-							absint( $result->ID )
-						), $verbose );
+					$this->assoc_args['uid_fields'] .= ',_customer_user';
+				}
+				// Explode and trim uid fields.
+				$uid_fields = array_filter( array_map( function($e) { return trim($e); }, explode( ',', $this->assoc_args['uid_fields'] )));
```

Comment:
To save an additional loop through the array, move `trim()` to inside the foreach and remove `array_map`
```
   foreach ( $uid_fields as $f ) {
        $f = trim( $f );
```